app/people.test.js

```diff
@@ -24,6 +24,37 @@ People.before(async (context) => {
   await sessionAuthPerform(tester, context)
 })
 
+People("invited a user", async (context) => {
+  const { perform } = inviteUser.operation
+  /** @type {InviteUserFields} */
+  const inputData = {
+    email: "whatever@onlyoffice.io",
+    type: "2"
+  }
+  const bundle = {
+    authData: context.authData,
+    inputData
+  }
+  try {
+    const user = await tester(perform, bundle)
+    if (!user) {
+      unreachable("TODO")
+      return
+    }
+    equal(user.displayName, bundle.inputData.email)
+  } catch (error) {
+    if (error instanceof Error && error.message) {
+      const response = JSON.parse(error.message)
+      if (response.content) {
+        const data = JSON.parse(response.content)
+        const message = data.error.message
+        equal(message, "User with this email is already registered")
+      }
+    }
```

Comment:
Perhaps it will be better this way.

```js
if (!(error instanceof Error && error.message)) {
  unreachable("was expecting an Error but got something else")
  return
}

const response = JSON.parse(error.message)
if (!!response.content) {
  unreachable("was expecting a content but did not get it")
  return
}

const data = JSON.parse(response.content)
const message = data.error.message
equal(message, "User with this email is already registered")
```

Reply:
> Perhaps it will be better this way.
> 
> ```js
> if (!(error instanceof Error && error.message)) {
>   unreachable("was expecting an Error but got something else")
>   return
> }
> 
> const response = JSON.parse(error.message)
> if (!!response.content) {
>   unreachable("was expecting a content but did not get it")
>   return
> }
> 
> const data = JSON.parse(response.content)
> const message = data.error.message
> equal(message, "User with this email is already registered")
> ```

It works, thanks!